source/blog/2015-06-05-ember-data-1-0-beta-19-released.md

```diff
@@ -43,6 +43,12 @@ Watson command](https://github.com/abuiles/ember-watson#ember-watsonconvert-embe
 You can use this command-line tool in both Ember-CLI projects and
 "globals mode" style projects.
 
+### Services depending on the store
+
+In order to fix deprecations warning induced by Ember 1.12, the store service is now injected as a
+instanceInitializer. As a consequence, if you had initializers depending on the store, you should move  
```

Comment:
@sly7-7 can you make `instanceInitializer` a hyperlink to http://emberjs.com/blog/2015/05/13/ember-1-12-released.html#toc_instance-initializers?